app/src/main/res/layout/item_horizontal_session.xml

```diff
@@ -113,12 +113,12 @@
 
             <io.github.droidkaigi.confsched2018.presentation.common.view.FavoriteButton
                 android:id="@+id/favorite"
+                android:layout_height="68dp"
```

Comment:
I have a question 🙋 
Why is this height needed? 🤔

Reply:
If I don't set height, it's `wrap_content`.
In that case, even if we have only one speaker, the height of `favorite` is lower than `speakers` .
So if we don't set this height, the vertical position of `favorite` and name of `speakers` are not same.
<img src="https://user-images.githubusercontent.com/10704349/35783189-895ab2de-0a46-11e8-94bb-92df43828cc9.png" width="300">

I think it looks not good, the vertical position of most bottom speaker and like button seems better to be same.

Reply:
Can we use constraintbottom_bottomOf="speaker" for favorite?